geospaas/base_viewer/templates/base_viewer/elements.html

```diff
@@ -1,77 +1,66 @@
 {% extends "base_viewer/page_structure.html" %}
 ##############################################################################################################
 {% block content_result %}
-<table border=1 width='100%' height='100%'>
-    <tr>
-        {% block headers %}
-        <th>Filename</th>
-        {% endblock headers %}
-    </tr>
+<table border=1 style="width:100%">
+  <tr>
+    {% block headers %}
+    <th>Datasets</th>
+    {% endblock headers %}
+  </tr>
 
-    {% if page_obj %}
+  {% if page_obj %}
 
-    {% for ds in page_obj %}
-    <tr class="dataset_row" ajax_url="{% url 'base_viewer:geometry_geojson' ds.geographic_location.pk %}" >
-        <td >
-            {% for url in ds.dataseturi_set.all %}
-            {% if show_local_address or url.service != local_file_service %}
-                {{ url.uri}} <br>
-            {% else %}
-              hidden local address of dataset <br>
-            {% endif %}
-            {% endfor %}
-        </td>
-
-    </tr>
+  {% for ds in page_obj %}
+  <tr class="dataset_row" ajax_url="{% url 'base_viewer:geometry_geojson' ds.geographic_location.pk %}">
+    <td>
+      {% block ds_info %}
+      {% endblock ds_info %}
+      {% for url in ds.dataseturi_set.all %}
+      {% if show_local_address or url.service != local_file_service %}
+      <div class="attr_title">Address:</div>
+      <div class="attr_content">{{ url.uri|escape|urlize}} </div>
+      {% else %}
+      <div class="attr_content">hidden local address of dataset</div>
+      {% endif %}
+      {% endfor %}
```

Comment:
Can you move this block with showing urls also to the ds_info.html ? 
Then it will be simpler to customize representation of Dataset on the web-page. 

For example, in adas-viewer we can either redefine `ds_info.html` and totally override it's content or we can define a new template and extend `base_viewer/ds_info.html` and add content.

Reply:
I thought that since it a code (I mean a for loop and other staff) we may dont want to override it and have this code in one place. This way the only limitation is that we can not define it in the middle of the informations, it should only be at the top or at the end of the other informations. Do you want the duplicate of it?